src/oic/__init__.py

```diff
@@ -3,9 +3,22 @@
 
 # Since SystemRandom is not available on all systems
 try:
-    import random.SystemRandom as rnd
+    # Python 3.6+, designed for this usecase
+    from secrets import choice
 except ImportError:
-    import random as rnd
+    import random
+    try:
+        # Python 2.4+ if available on the platform
+        _sysrand = random.SystemRandom()
+        choice = _sysrand.choice
+    except AttributeError:
```

Comment:
I don't think that we officially support `python < 2.7` so I think this `except` branch is not really necessary.

Reply:
True, but even on Python 2.7 the docs state that some platforms lack SystemRandom().

Not that it would be a good idea to deploy this code on any such platform...

Reply:
OK